### PR TITLE
Add common cmd+k use cases

### PR DESCRIPTION
**Pull request by janine-c**
### What does this PR do? What is the motivation?

Adding some common examples of when to use `cmd` + `k` to open the global search.

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/getting_started/application/_index.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ This page provides a high-level overview of capabilities available on the [Datad
 
 The Datadog site navigation varies based on the width of your browser. You can have up to three types of navigation. To change the navigation type, adjust your browser width.
 
+**Tip**: You can press <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>K</kbd> anytime to search for pages, entities, and common actions across Datadog.
```
Comment (michaelcretzman) [inline content/en/getting_started/application/_index.md:31]:
```suggestion
**Tip**: You can press <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>K</kbd> to search for pages, entities, and common actions across Datadog.
```

File: content/en/logs/_index.md [hunk 2]
```diff
@@ -88,6 +88,8 @@ Leverage the pillars of observability by connecting your logs to metrics and tra
 
 Start exploring your ingested logs in the [Log Explorer][1].
 
+**Tip**: To get to the Log Explorer quickly, press <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>K</kbd> to open Datadog's global search, then search for `logs`.
```
Comment (michaelcretzman) [inline content/en/logs/_index.md:91]:
```suggestion
**Tip**: To open the Log Explorer from Datadog's global search, press <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>K</kbd> and search for `logs`.
```

Comment (michaelcretzman) [reply content/en/logs/_index.md:91]: just suggesting a different phrasing. Feel free to ignore it or use it elsewhere.

**Review by michaelcretzman — APPROVED** [review]
approved with a minor suggestion

**Comment by kdobesh** [comment]
- https://docs-staging.datadoghq.com/janine.chan/docs-9287-cmd-k-search/getting_started/application/:
the copy says "...pages, entities, and common actions..." can we remove the "actions" part? cuz I think we actually have an entity called actions, and you cannot get those in cmdK right now. i'd also like to give an example of an entity in the copy too, so maybe saying "Tip: You can press Cmd/Ctrl + K to search for pages and entities like dashboards & monitors across Datadog."? 

- https://docs-staging.datadoghq.com/janine.chan/docs-9287-cmd-k-search/metrics/
I'm worried that saying "to open your metrics from DD's global search, press..." will make people think they can search for distinct metrics within cmdK, which you also cannot do. Maybe say "to view a list of all your metrics from DD's gsearch.." or "to open the Metrics Summary page from DD's gsearch..." would be clearer?

- https://docs-staging.datadoghq.com/janine.chan/docs-9287-cmd-k-search/service_management/events/
so right now, if you type "Events" in cmdK, the top result will open not a list of individual events, but rather GROUPS of similar events (aka the Correlated Events view, which is new and not well adopted). if users want to see a list of distinct events, they need to type "Event Explorer" so maybe change that? 

everything else was great though! thank you so much for the speedy turn around on this!

**Comment by janine-c** [comment]
Thank you for your thorough and valuable feedback, @kdobesh ! I've applied all of it; please take another look 🙂

**Review by kdobesh — APPROVED** [review]
looks great @janine-c thank you! Tim is OOO until Monday, but I think his will be minimal since we've been closely aligned throughout this project. nevertheless, we are going to wait til Monday to send our announcement email assuming that merging these docs won't take long after he approves it?

**Review by brimtown — APPROVED** [review]

**Pull request merged**